magni_bringup/param/robot.yaml

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
This should be default_robot.yaml. It should probably also move to a new `config/` directory instead of being in `/param`.

We can probably rely on this file existing because it is in the package.